tests/test_reqif.py

```diff
@@ -639,6 +650,87 @@ def test_create_RequirementType_AttributeDefinition_creation(
+        self, model: capellambse.MelodyModel
+    ):
+        reqtypesfolder = model.by_uuid("67bba9cf-953c-4f0b-9986-41991c68d241")
+        dt_definitions = reqtypesfolder.data_type_definitions
+
+        edt_def = reqtypesfolder.data_type_definitions.create(
+            "EnumerationDataTypeDefinition",
+            long_name="Enum",
+        )
+        edt_def.values.create(long_name="val")
+        edt_def.values.create(long_name="val1")
+
+        assert len(dt_definitions) + 1 == len(
+            reqtypesfolder.data_type_definitions
+        )
+        assert edt_def in reqtypesfolder.data_type_definitions
+        assert set(edt_def.values.by_long_name) == {"val", "val1"}
+
+    def test_create_RequirementTypesFolder_EnumDataTypeDefinition_creating_EnumValues(
+        self, model: capellambse.MelodyModel
+    ):
+        reqtypesfolder = model.by_uuid("67bba9cf-953c-4f0b-9986-41991c68d241")
+        dt_definitions = reqtypesfolder.data_type_definitions
+
+        edt_def = reqtypesfolder.data_type_definitions.create(
+            "EnumerationDataTypeDefinition",
+            long_name="Enum",
+            values=[{"long_name": "val"}, {"long_name": "val1"}],
+        )
+
+        assert len(dt_definitions) + 1 == len(
+            reqtypesfolder.data_type_definitions
+        )
+        assert edt_def in reqtypesfolder.data_type_definitions
+        assert set(edt_def.values.by_long_name) == {"val", "val1"}
+
+    def test_create_RequirementTypesFolder_EnumDataTypeDefinition_creating_EnumValues_with_singleattr(
+        self, model: capellambse.MelodyModel
+    ):
+        reqtypesfolder = model.by_uuid("67bba9cf-953c-4f0b-9986-41991c68d241")
+        dt_definitions = reqtypesfolder.data_type_definitions
+
+        edt_def = reqtypesfolder.data_type_definitions.create(
+            "EnumerationDataTypeDefinition",
+            long_name="Enum",
+            values=["val", "val1"],
+        )
+
+        assert len(dt_definitions) + 1 == len(
+            reqtypesfolder.data_type_definitions
+        )
+        assert edt_def in reqtypesfolder.data_type_definitions
+        assert set(edt_def.values.by_long_name) == {"val", "val1"}
+
+    def test_create_RequirementTypesFolder_from_mapping(
+        self, model: capellambse.MelodyModel
+    ):
+        reqtypesfolder = model.la.requirement_types_folders.create(
+            long_name="Test",
+            data_type_definitions=[
+                {
+                    "type_hints": "DataTypeDefinition",
+                    "long_name": "TestAttrDataTypeDef",
+                },
+                {
+                    "type_hints": "EnumerationDataTypeDefinition",
+                    "long_name": "TestEnumAttrDataTypeDef",
+                    "values": ["a", "b"],
+                },
+            ],
+        )
+
+        adtdef, edtdef = reqtypesfolder.data_type_definitions
+
+        assert isinstance(adtdef, reqif.DataTypeDefinition)
```

Comment:
The spacing makes it seem as if the `adtdef, edtdef = reqtypesfolder.data_type_definitions` is the line-under-test, when it's actually the `create` statement above. Judging from the test case name, this line actually belong the assertions though, and should not be separated from them by an empty line.